core/rawdb/accessors_state_arbitrum.go

```diff
@@ -22,15 +22,76 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
-// Stores the activated asm and module for a given codeHash
-func WriteActivation(db ethdb.KeyValueWriter, moduleHash common.Hash, asm, module []byte) {
-	key := ActivatedAsmKey(moduleHash)
+const (
+	TargetWavm = "wavm"
+	TargetArm  = "arm"
+	TargetX86  = "x86"
```

Comment:
not 100% certain - but I think maybe TargetArm and TargetAsm should be identical to the runtime.GOARCH on arm64/x86 architectures, which will be compatible with some exiting code

Reply:
I used GOARCH strings + I renamed the constants to TargetArm64 and TargetAmd64 to make it more consistent